apps/opik-backend/src/main/java/com/comet/opik/domain/FeedbackScoreDAO.java

```diff
@@ -315,36 +210,43 @@ private FeedbackScoreDto mapFeedback(Row row) {
     public Mono<Long> scoreEntity(@NonNull EntityType entityType,
             @NonNull UUID entityId,
             @NonNull FeedbackScore score,
+            @NonNull Map<UUID, UUID> entityProjectIdMap,
             @NonNull Connection connection) {
 
-        var template = new ST(INSERT_FEEDBACK_SCORE);
-
-        Optional.ofNullable(score.categoryName())
-                .ifPresent(category -> template.add("categoryName", category));
-
-        Optional.ofNullable(score.reason())
-                .ifPresent(comment -> template.add("reason", comment));
+        var template = new ST(BULK_INSERT_FEEDBACK_SCORE);
 
-        template
-                .add("entity_table", entityType.getTableName());
+        List<FeedbackScore> scores = List.of(score);
+        List<TemplateUtils.QueryItem> queryItems = getQueryItemPlaceHolder(scores.size());
 
-        var statement = connection.createStatement(template.render())
-                .bind("entity_type", entityType.getType())
-                .bind("name", score.name())
-                .bind("value", score.value().toString())
-                .bind("entity_id", entityId)
-                .bind("source", score.source().getValue());
+        template.add("items", queryItems);
 
-        Optional.ofNullable(score.reason())
-                .ifPresent(comment -> statement.bind("reason", comment));
+        var statement = connection.createStatement(template.render());
```

Comment:
You broke the method into smaller chunks: bindParameters, getBatchSql etc. but my ask was to just call `scoreBatchOf` from `scoreEntity`, otherwise you're still duplicating logic. This is actually and overloaded method, they just have different names. Some pseudocode:
```
  public Mono<Long> scoreEntity(@NonNull EntityType entityType,
            @NonNull UUID entityId,
            @NonNull FeedbackScore score,
            @NonNull Map<UUID, UUID> entityProjectIdMap) {
   var batchScore = FeedbackScoreMapper.toFeedbackScoreBatchItem(...)
   return scoreBatchOf(entityType, List.of(batchScore));
}
```

Anyway, this is not a blocker.

Reply:
Ok, I got it. I didn't understand it because the objects were different. Ok, let me try to map it because there is also the project field